conf/documentation.conf

```diff
@@ -1576,3 +1576,31 @@ Which changes (changing from a device class to another) should trigger internal:
 This setting is independant from trigger_on_device_class_change and allows to specify exactly which transitions should trigger internal::fingerbank_device_change. 
 Comma delimited transitions using the following format: $PREVIOUS_DEVICE_CLASS_ID->$NEW_DEVICE_CLASS_ID where $PREVIOUS_DEVICE_CLASS_ID and $NEW_DEVICE_CLASS_ID are the IDs in the Fingerbank database.
 EOT
+
+[lets_encrypt.radius]
+type=toggle
+options=enabled|disabled
+description=<<EOT
+Whether or not Let's Encrypt should be enabled for RADIUS certificate management
```

Comment:
certificate -> certificates ? Same below.

Reply:
There is just one certificate for RADIUS unless I'm getting my grammar wrong

Reply:
So, final verdict, should it be certificate or certificates ?